hack/ci/setup-env.sh

```diff
@@ -7,6 +7,8 @@ export NAME="stage"
 # Remember when changing LOAD_BALANCER_EXTERNAL_IP to update record A in the Cloud DNS for gateway
 cat <<EOT >> "$GITHUB_ENV"
 GO_VERSION=^1.16.3
+GOLANGCI_LINT_VERSION=v1.41.1
```

Comment:
I thought also about adding these variables to `const.sh and calling it here + in the `lint.sh` script, but I am not sure if these variables are a good fit for that file. I would keep it that way but I am open to suggestions.

Reply:
I'm ok with that approach for now 👍